src/output/icons.rs

```diff
@@ -283,11 +306,13 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     "requirements.txt"    => Icons::LANG_PYTHON,    // 
     "robots.txt"          => '\u{f06a9}',           // 󰚩
     "rubydoc"             => Icons::LANG_RUBYRAILS, // 
+    "rustfmt.toml"        => Icons::LANG_RUST,      // 
     "rvmrc"               => Icons::LANG_RUBY,      // 
     "settings.gradle.kts" => Icons::GRADLE,         // 
     "shadow"              => Icons::LOCK,           // 
     "shells"              => Icons::CONFIG,         // 
     "sudoers"             => Icons::LOCK,           // 
+    "template"            => '\u{f32e}',            // 
```

Comment:
this one could be matched on the `.template` files too

Reply:
Composer is the unofficial package manager for PHP. Maybe `composer.json` appears in non-PHP projects sometimes, but not often.